src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -157,36 +149,37 @@ private void ProcessCommPkgEvent(string messageJson)
             }
 
             _plantSetter.SetPlant(commPkgEvent.Plant);
-            if (!string.IsNullOrWhiteSpace(commPkgEvent.ProjectNameOld))
-            {
-                _telemetryClient.TrackEvent(IpoBusReceiverTelemetryEvent,
-                    new Dictionary<string, string>
-                    {
-                        {PcsServiceBusTelemetryConstants.Event, IpoTopic.TopicName},
-                        {PcsServiceBusTelemetryConstants.CommPkgNo, commPkgEvent.CommPkgNo},
-                        {PcsServiceBusTelemetryConstants.Plant, commPkgEvent.Plant[4..]},
-                        {PcsServiceBusTelemetryConstants.ProjectName, commPkgEvent.ProjectName.Replace('$', '_')},
-                        {PcsServiceBusTelemetryConstants.ProjectNameOld, commPkgEvent.ProjectNameOld.Replace('$', '_')}
-                    });
-                _invitationRepository.MoveCommPkg(
-                    commPkgEvent.ProjectNameOld,
-                    commPkgEvent.ProjectName,
-                    commPkgEvent.CommPkgNo,
-                    commPkgEvent.Description);
-            }
-            else
+
+
+            // Get or create project
+            var project = _invitationRepository.GetProject(commPkgEvent.ProjectName);

```

Comment:
Is it correct that new code from here, getting or creating a project, will handle a move of a commpkg from project to project? If so, add a comment saying so
An unit test should be added, where you mock that _invitationRepository.GetProject return null. The unit test should then assert that _projectApiService.TryGetProjectAsync is called once. Also assert that _projectRepository.Add a project (I can assist howto on such test)